lib/medtronic/medtronicSimulator.js

```diff
@@ -222,19 +266,25 @@ exports.make = function(config){
             };
             event.suppressed = suppressed;
 
+            if(!event.rate) {
+              event = event.with_rate(currBasal.rate * event.percent);
```

Comment:
Worth doing a check here first to see if `event.percent` exists? And throw an error if it's not there? It won't exist if the `temp` is manual, I imagine...